src/module/item/npc-class-sheet.ts

```diff
@@ -85,138 +85,6 @@ export class LancerNPCClassSheet extends LancerItemSheet<EntryType.NPC_CLASS> {
     return data.map(x => parseFloat(x));
```

Comment:
Removed ability to add features to classes--should re-add for #110

Reply:
Oh, right. You still can, I suppose it's not as intuitive anymore though

Reply:
How can you? Haven't been able to figure it out... is my world maybe just broken for dragdrop?

Reply:
Drag the portrait of the feature, which acts as a regret draggable and thus has the proper logic to reveal the drop locations. I've ideas for how to solve this issue but need to avoid orphaned listeners

Reply:
Ah... yeah that's weird, no easy way to tie that into the compendiums I'm guessing based on current setup? That's how it'll be done 99% of the time so that's the one we'll need to have work